package.json

```diff
@@ -59,5 +59,10 @@
     "rewire": "^5.0.0",
     "sinon": "^7.3.2",
     "sinon-chai": "^3.3.0"
+  },
+  "mocha": {
+    "recursive": true,
+    "require": "test/common.js",
+    "reporter": "spec"
```

Comment:
I don't think we should make this change unless we are going to make it everywhere.

I have a feeling that the existing approach has it's benefits in other places. Probably lev-web. (Do integration tests get their own config?)

Reply:
The existing approach is deprecated, hence this change, so that's not really an option. There is supposed to be a `.mocharc` alternative, but that doesn't seem to work, so `package.json` FTW.

Reply:
I'll remove this for the moment then, and it can go on the backlog as tech debt I suppose. Fixed

Reply:
Okay.. Sounds good. It would be good to have a link to it being deprecated and what they suggest as the best alternative.

I suspect we will end up replacing mocha with jest anyway so this issue will probably go away.